examples/centralized_design/example.tfvars

```diff
@@ -176,109 +176,108 @@ vpcs = {
       }
     }
     subnets = {
-      # Do not modify value of `set=`, it is an internal identifier referenced by main.tf
-      "10.100.0.0/24"  = { az = "eu-west-1a", set = "mgmt", nacl = null }
-      "10.100.64.0/24" = { az = "eu-west-1b", set = "mgmt", nacl = null }
-      "10.100.1.0/24"  = { az = "eu-west-1a", set = "private", nacl = "trusted_path_monitoring" }
-      "10.100.65.0/24" = { az = "eu-west-1b", set = "private", nacl = "trusted_path_monitoring" }
-      "10.100.2.0/24"  = { az = "eu-west-1a", set = "public", nacl = null }
-      "10.100.66.0/24" = { az = "eu-west-1b", set = "public", nacl = null }
-      "10.100.3.0/24"  = { az = "eu-west-1a", set = "tgw_attach", nacl = null }
-      "10.100.67.0/24" = { az = "eu-west-1b", set = "tgw_attach", nacl = null }
-      "10.100.4.0/24"  = { az = "eu-west-1a", set = "gwlbe_outbound", nacl = null }
-      "10.100.68.0/24" = { az = "eu-west-1b", set = "gwlbe_outbound", nacl = null }
-      "10.100.5.0/24"  = { az = "eu-west-1a", set = "gwlb", nacl = null }
-      "10.100.69.0/24" = { az = "eu-west-1b", set = "gwlb", nacl = null } # AWS reccomends to always go up to the last possible AZ for GWLB service
-      "10.100.10.0/24" = { az = "eu-west-1a", set = "gwlbe_eastwest", nacl = null }
-      "10.100.74.0/24" = { az = "eu-west-1b", set = "gwlbe_eastwest", nacl = null }
-      "10.100.6.0/24"  = { az = "eu-west-1a", set = "alb", nacl = null }
-      "10.100.70.0/24" = { az = "eu-west-1b", set = "alb", nacl = null }
-      "10.100.7.0/24"  = { az = "eu-west-1a", set = "nlb", nacl = null }
-      "10.100.71.0/24" = { az = "eu-west-1b", set = "nlb", nacl = null }
+      "10.100.0.0/24"  = { az = "eu-west-1a", subnet_group = "mgmt", nacl = null }
```

Comment:
I assume similar approach will be used for other examples in this PR or it's going to be in separated PR ?

Reply:
Yes, the same approach will be used for all examples